Models/biophysics_calibration/pyabc_cellmodeller.py

```diff
@@ -55,42 +101,73 @@ def sim_time_parameters(file_path):
     Must be written as dt = X.Y in the module file
     
     @param  file_path   str of file name containing parameters to be modified
+    @return dt          Simulation time step specified in CellModeller module file
     """
     search_dt = f'dt = (\d+\.\d+)'
     with open(file_path, "r+") as file:
         file_contents = file.read()
         dt = re.findall(search_dt, file_contents)
     return float(dt[0])
+       
+def distance_calculation(sim_stats, exp_stats):
```

Comment:
This function was written by Jonathan and edited by me. It would have been better to mention it like the way I mentioned in my codes

Reply:
We should compare notes here. I don't think we're come to a conclusion on the best approach.

Reply:
@bpingalls yes, I'm unsure about what the pros/cons are for various distance metrics. Would be good to discuss.
@Ati-74 It seems like pyabc already has a built-in [Euclidean distance function](https://pyabc.readthedocs.io/en/latest/api_distance.html?highlight=pnorm#pyabc.distance.PNormDistance), which seems like a more elegant solution. I already had this implemented in my previous [commit](https://github.com/ingallslab/CellModeller-ingallslab/commit/54c68e2a20b0095740b54a473a1b1c83786ffd18), but changed it here at your request. I will chat with @bpingalls about distance metrics and take it from there.